Detectors/MUON/MCH/Simulation/src/Response.cxx

```diff
@@ -48,46 +82,39 @@ float Response::etocharge(float edepos)
   return charge;
 }
 //_____________________________________________________________________
-double Response::chargePad(float xmin, float xmax, float ymin, float ymax, int detID, float charge)
+double Response::chargePad(float xmin, float xmax, float ymin, float ymax, float charge)
```

Comment:
here I would remove the charge (which is only a scaling factor in the end), and maybe make it more apparent (1 function doing the integration in 1D and this one calling it twice with different params?) it's really IntX * IntY that is computed here.

Reply:
Yes indeed, it is just the product of the two dimensions. 
I changed now the implementation.